src/components/pages/devices/devices.js

```diff
@@ -12,18 +12,12 @@ export class Devices extends Component {
     this.props.fetchDevices();
   }
 
-  renderDevices(devices, noDevicesMessage) {
-    return devices && devices.length
-      ? <DeviceGrid rowData={devices} />
-      : noDevicesMessage;
-  }
-
   render() {
-    const { t, devices, error, fetchDevices } = this.props;
-
+    const { t, devices, error, isPending, fetchDevices } = this.props;
     return (
       <div className="devices-container">
-        { this.renderDevices(devices, error ? error : t('devices.noneFound')) }
+        { !!error && <span className="status">Error: {error.errorMessage}</span> }
```

Comment:
Need to translate "Error"

Reply:
Yep, will do a sweep for these a later PR.